src/components/editor/AppController.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { browserHistory } from 'react-router';
+import FalcorController from "lib/falcor/FalcorController";
+import { setAppReady } from "lib/falcor/falcorUtils";
+
+export default class AppController extends FalcorController {
+  constructor(props) {
+    super(props);
+    this.safeSetState({
+      defaultValue: this.handleDefaultValue.bind(this)()
+    })
+    
+  }
+  static getFalcorPath() {
+    return;
+  }
+
+  componentDidMount() {
+    super.componentDidMount();
+    setAppReady();
+  }
+
+  handleDefaultValue() {
+    let path = this.props.location.pathname;
+    if (path === '/') {
+      return '/';
+    }
+    else if (path[2] === 'r') {
+      return 'articles';
+    }
+    else {
+      return 'authors';
+    }
+  }
+
+  handleChange(event) {
+    if (event.target.value === '/') {
+      browserHistory.push("/");
```

Comment:
Yeah I think I'm starting to understand where you're coming from, I'll mess around it with it later and show you what I come up with and see if you like it better. 

I'll probably change a few things, so you can also look at it then, but the difference between / and /authors and /articles is / doesn't show anything below it, it's just the blank welcome page.